### PR TITLE
Adjusted Creator Hub Screen according to suggested changes.

### LINKED ISSUE #1280: EASEL Creator Hub Screen Adjustments
## Summary
Need to adjust UI changes in creator hub

## Proposal

Screen_Shot_2022-10-20_at_3 05 08_PM

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by Kamran-Khan-Gene**
Updated Creator Hub Screen and Implemented suggested changes.
closes #1280

### CODE REVIEW

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  kindly add the description.

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  Kindly add screenshots of android, ios and tablet.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart [hunk 1]
```diff
@@ -389,8 +353,8 @@ class NFTsViewBuilder extends StatelessWidget {
 class BuildGridView extends StatelessWidget {
   final List<NFT> nftsList;
   final WidgetBuilder onEmptyList;
-
-  const BuildGridView({Key? key, required this.nftsList, required this.onEmptyList}) : super(key: key);
+  final CollectionType collectionType;
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/creator_hub_screen.dart:356]: Whats the reason for add this as a part of constructor. We can get it through provider in the NFTGridViewItem without passing,

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Comment by Kamran-Khan-Gene** [comment]
> @Kamran-Khan-Gene Kindly add screenshots of android, ios and tablet.

Here are these screenshots from Android Phone.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart (hunk 1 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart:356]: Ok, I am updating it now. Thank you

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  Need following changes.
1) `There are no drafts should not touch the left side of the screen`. 
2) Create Draft shouldn't touch the left and right screen.

**Comment by kjawadDeveloper2** [comment]
@StefanB90 @Tanyaml  kindly review the made designs.

**Comment by Kamran-Khan-Gene** [comment]
> @Kamran-Khan-Gene Need following changes.
> 
> 1. `There are no drafts should not touch the left side of the screen`.
> 2. Create Draft shouldn't touch the left and right screen.

@kjawadDeveloper2  I did attached old screenshot mistakenly... I have verified it now. I am attaching updating screen shot.

Screenshot_20221025_212943

File: easel/lib/screens/creator_hub/creator_hub_screen.dart (hunk 1 above)
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/creator_hub_screen.dart:356]: I can still see collection type here.

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart [hunk 2]
```diff
@@ -58,6 +62,47 @@ class NftGridViewItem extends StatelessWidget {
...
+              end: Alignment.bottomCenter,
+            ),
+          ),
+          child: Row(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(1.h),
+                  color: context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft ?EaselAppTheme.kLightRed:
+                  nft.isEnabled && nft.amountMinted < int.parse(nft.quantity)?EaselAppTheme.kBlue:EaselAppTheme.kDarkGreen,
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:79]: Kindly format this screen. Looks like this line is not formatted.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  can you add test also?

File: easel/lib/screens/creator_hub/creator_hub_screen.dart (hunk 1 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart:356]: I have removed this the collection type variable, and committed code 12 hours ago.. I think I am unable to merge my this commit into pull request.


image

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart (hunk 2 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:79]: Sorry i did not get your point, can you please elaborate? format how?

**Comment by StefanB90** [comment]
Just finished this with @xinya & @MikeSofaer:
image

**Comment by Tanyaml** [comment]
@StefanB90 is there any way you can move the draft and published tabs more center and move the display icons to the left?

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart (hunk 2 above)
Comment (faisalnaveedRNS) [reply easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:79]: I think we need to format the whole file

Comment (faisalnaveedRNS) [reply easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:79]: @Kamran-Khan-Gene you need to adjust max line, to 200 char in your editor

**Comment by Kamran-Khan-Gene** [comment]
@kjawadDeveloper2 I have incorporated updated design changes, kindly review my changes.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart [hunk 3]
```diff
@@ -407,7 +348,19 @@ class BuildGridView extends StatelessWidget {
...
-          return NftGridViewItem(nft: nft);
+          return (nft.price.isNotEmpty && nft.price != "0")
+              ? ClipRRect(
+                  child: Banner(
+                      color: EaselAppTheme.kDarkGreen,
+                      location: BannerLocation.topEnd,
+                      message: "\$ ${nft.price}",
+                      child: NftGridViewItem(
+                        nft: nft,
+                      )),
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/creator_hub_screen.dart:359]: Formatting not done.

File: easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart [hunk 4]
```diff
@@ -58,6 +65,72 @@ class NftGridViewItem extends StatelessWidget {
...
+                        final DraftsBottomSheet draftsBottomSheet = DraftsBottomSheet(
+                          buildContext: context,
+                          nft: nft,
+                        );
+                        draftsBottomSheet.show();
+                      },
+                      child: Padding(
+                        padding: EdgeInsets.all(4.0.w),
+                        child: SvgPicture.asset(SVGUtils.kSvgMoreOption, color: Colors.white),
+                      )),
```
Comment (kjawadDeveloper2) [inline easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart:125]: Formatting not done.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]

File: easel/lib/screens/creator_hub/creator_hub_screen.dart [hunk 5]
```diff
@@ -407,7 +348,19 @@ class BuildGridView extends StatelessWidget {
         ),
         itemBuilder: (context, index) {
           final nft = nftsList[index];
-          return NftGridViewItem(nft: nft);
+          return (nft.price.isNotEmpty && nft.price != "0")
```
[the remark this answers is not in the document]
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart:351]: Ok I will do it.

File: easel/lib/screens/creator_hub/widgets/draft_list_tile.dart [hunk 6]
```diff
@@ -132,68 +199,20 @@ class _DraftListTileState extends State<DraftListTile> {
...
-                ),
-              ]),
-              child: Padding(
-                padding: EdgeInsets.symmetric(horizontal: 15.w, vertical: 15.h),
-                child: Row(
-                  children: [
-                    buildAssetView(),
-                    SizedBox(
-                      width: 10.w,
+          child: (widget.nft.price.isNotEmpty && widget.nft.price != "0")
```
[the remark this answers is not in the document]
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/widgets/draft_list_tile.dart:202]: Ok I will do it.. thank you for correcting it.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart [hunk 7]
```diff
@@ -98,35 +98,6 @@ class _CreatorHubContentState extends State<CreatorHubContent> {
 
   EaselProvider get easelProvider => sl();
 
-  void onRefreshPressed() async {
```
[the remark this answers is not in the document]
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart:101]: sure

**Comment by kamran8545** [comment]
@kjawadDeveloper2 kindly review my PR.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart:372]: @Kamran-Khan-Gene  whats the update on this?

**Comment by kjawadDeveloper2** [comment]
@Kamran-Khan-Gene  there is one unresolved comment it should be done by now.

File: easel/lib/screens/creator_hub/creator_hub_screen.dart (hunk 1 above)
Comment (Kamran-Khan-Gene) [reply easel/lib/screens/creator_hub/creator_hub_screen.dart]: I'ts done.

**Comment by Kamran-Khan-Gene** [comment]
> @Kamran-Khan-Gene there is one unresolved comment it should be done by now.

It is done now.. Kindly review my PR.

**Comment by Kamran-Khan-Gene** [comment]
@kjawadDeveloper2 kindly review my PR.

**Review by kjawadDeveloper2 — APPROVED** [review]
LGTM. Just want to see the screenshots of the tablet.

**Comment by kamran8545** [comment]
Here the updated screen shots.


Screenshot_20221103_205332


Screenshot_20221103_205337



Screenshot_20221103_205342




Screenshot_20221103_205348

**Review by Amean — APPROVED** [review]

**Review by faisalnaveedRNS — APPROVED** [review]

**Pull request merged**